plugins/recommendations-plugin/tsconfig.json

```diff
@@ -1,33 +1,19 @@
 {
+    "extends": "../../configs/base.tsconfig",
     "compilerOptions": {
-      "target": "es6",
-      "module": "commonjs",
-      "moduleResolution": "node",
-      "outDir": "./lib",
-      "sourceMap": true,
-      "noEmitOnError": true,
-      "noImplicitReturns" : true,
-      "noImplicitThis": true,
-      "noUnusedLocals": false,
-      "removeComments": true,
-      "noFallthroughCasesInSwitch": true,
-      "strict": true,
-      "strictPropertyInitialization": false,
-      "skipLibCheck": true,
-      "skipDefaultLibCheck": true,
-      "types": ["node", "reflect-metadata", "jest"],
-      "experimentalDecorators": true,
-      "emitDecoratorMetadata": true,
-      "lib": [
-        "es6"
-    ],
-    }
-  ,
+        "target": "es5",
```

Comment:
The other plugins are using "es6" or nothing at all (inherited from the base config) for a target.

Reply:
Well I copy it from containers-plugin:
There are many other references https://github.com/eclipse-che/che-theia/search?q=es5
And it'll be replaced anyway by https://github.com/eclipse-che/che-theia/pull/1213 so it's like uneeded requirement